osbs/build/build_request.py

```diff
@@ -562,6 +562,7 @@ def render_add_labels_in_dockerfile(self):
             'vendor': self.spec.vendor,
             'authoritative-source-url': self.spec.authoritative_registry,
             'distribution-scope': self.spec.distribution_scope,
+            'release': self.spec.release,
```

Comment:
We shouldn't set `'release'` here unless `self.spec.release` is truthy. In other words, we don't want to override the release label with a value unless we have a value to override it with.